packages/ui/src/Table/Row.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from 'react'
+import cn from 'classnames'
+
+import styles from './Row.module.scss'
+import { TableHeaderGroupProps } from 'react-table'
+
+type RowClickProps =
+  | {
+      isHeaderRow: false
+      onClick?: () => void
+    }
+  | {
+      isHeaderRow: true
+      onClick?: never
+    }
+
+export type RowProps = RowClickProps & Omit<TableHeaderGroupProps, 'key'> & { ariaRowIndex?: number }
+
+export const Row: FC<RowProps> = ({ children, isHeaderRow = false, onClick, className, style, role, ariaRowIndex }) => {
+  if (isHeaderRow) {
+    return (
+      <div data-test="table-header-row" className={cn(styles.headerRow, className)} style={style} role={role} aria-rowindex={ariaRowIndex}>
+        {children}
+      </div>
+    )
+  }
+
+  return (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events
+    <div
```

Comment:
Do you think we could try adding `rowHref` prop of `(rowInfo: RowType<D>) => string` and make `Row` an `a` if  `rowHref` is present? I, personally, like clicking on the entire row instead of trying to find a clickable item inside of it. WDYT?
We could allow devs specify either `onRowClick` or `rowHref`.

Reply:
That's worth a shot. Let me try this approach!

Reply:
`Table` now has two mutually exclusive props: `onClick` and `getHref`
```
onRowClick?: (rowInfo: RowType<D>) => void
getHref?: (rowInfo: RowType<D>) => string
```

`getHref` has to return string which is then used as `href` in our `<a>` element.